hasura/migrations/default/1656931867198_create_table_public_contributions/up.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE "public"."contributions" ("id" serial NOT NULL, "epoch_id" integer NOT NULL, "user_id" integer NOT NULL, "description" text, "deleted_at" timestamptz, "created_at" timestamptz NOT NULL DEFAULT now(), "updated_at" timestamptz NOT NULL DEFAULT now(), PRIMARY KEY ("id") , FOREIGN KEY ("epoch_id") REFERENCES "public"."epoches"("id") ON UPDATE restrict ON DELETE restrict, FOREIGN KEY ("user_id") REFERENCES "public"."users"("id") ON UPDATE restrict ON DELETE restrict, UNIQUE ("id"));
```

Comment:
can you edit this file so each column is on its own line? so that it's easier to read & we can comment on individual fields